src/components/GsocTsoc/GsocTsoc.jsx

```diff
@@ -73,41 +73,43 @@ const GsocTsoc = () => {
     <div>
       <div>
         <h4 className="text-center">
-          About Google summer of code & Terasology summer of code{" "}
+          {`About Google summer of code & Terasology summer of code${" "}`}
```

Comment:
Why do we need both here, the backtick escape but still the curly braces with a space?